docs/api-restful.rst

```diff
@@ -33,16 +33,16 @@ Synopsis
 .. code-block:: bash
 
     # Download a main dataset JSON file…
-    curl https://nextstrain.org/seasonal-flu/h3n2/ha/2y                         \
+    curl https://nextstrain.org/zika                         \
         --header 'Accept: application/vnd.nextstrain.dataset.main+json'         \
         --compressed                                                            \
-            > seasonal-flu_h3n2_ha_2y.json
+            > zika.json
```

Comment:
I also contemplated replacing with another example that uses the `/` to `_` pattern, but realized that's not a feature relevant to this API and more for something like [`nextstrain remote download`](https://docs.nextstrain.org/projects/cli/en/8.5.0/commands/remote/download/) where that's clearly documented already.

Reply:
Hmm. I used seasonal-flu exactly for this reason: to highlight how multi-part pathogen URLs work and to show by example the local filesystem naming convention (even if you can ignore it, you're likely better off not). I'd suggest switching to mpox or ncov/open or something similar where we can maintain the multi-part example.

Reply:
Done: https://github.com/nextstrain/nextstrain.org/pull/946